DLCS.HydraModel/DlcsResource.cs

```diff
@@ -68,8 +68,8 @@ public virtual JObject GetCollectionForm()
 
         public override string Context =>
             // do this better later - prefer not have FullName but makes reflection easier!
-            BaseUrl + "/contexts/" + GetType().FullName + ".jsonld";
+            BaseUrl + "/contexts/" + GetType().Name + ".jsonld";
```

Comment:
Use string interpolation here for consistency?

Reply:
Yeah a lot of Hydra code here from 2016, will fix up